src/test/resources/features/cs/ManagementUI.feature

```diff
@@ -454,9 +489,28 @@ Feature: ManagementUI
 
 #    MANAGE MPAM BUSINESS UNITS
 
+  @ListsManagement
+  Scenario: User is able to create new business units through Lists management
+    When I select to "Manage MPAM Business Units"
+    And I load the business units for the "UKVI" business area
+    And I create a new business unit
+    Then a success message is displayed
+    And I select to "Manage MPAM Business Units"
+    And I load the business units for the "UKVI" business area
+    Then the new business unit is added to the list of business units
+
 
 #    MANAGE MPAM ENQUIRY REASONS
 
+  @ListsManagement
+  Scenario: User is able to create new enquiry reason through Lists management
+    When I select to "Manage MPAM Enquiry Reasons"
+    And I load the enquiry reasons for the "Person Specific" enquiry subject
+    And I create a new enquiry reason
+    Then a success message is displayed
+    And I select to "Manage MPAM Enquiry Reasons"
+    And I load the enquiry reasons for the "Person Specific" enquiry subject
```

Comment:
Same with these 2 steps. Lets move them into the glue of the bottom assertion, it will read better that way.